guides/common/modules/proc_preparing-for-disaster-recovery-with-two-active-project-servers.adoc

```diff
@@ -0,0 +1,51 @@
+[id="preparing-for-disaster-recovery-with-two-active-project-servers"]
+= Preparing for disaster recovery with two active {ProjectServer}s
+
+Create a second {ProjectServer} by restoring a backup of your first {ProjectServer}.
+Configure both servers to operate independently in their respective data centers, but ensure that their content, such as repositories or host configuration, does not drift apart over time.
```

Comment:
We might also want to stick with "content" only. We get a little bit more specific in the procedure below and based on that, I'd say "content" is enough here.

```suggestion
Configure both servers to operate independently in their respective data centers, but ensure that their content does not drift apart over time.
```